src/js/profile/chart.js

```diff
@@ -237,7 +242,43 @@ export class Chart extends Observable {
       ],
     };
 
-    embed(this.container, barChart, { actions: false})
+
+    var handler = (h, event, item, value) => {
+      this.el = document.getElementById(DEFAULT_OPTIONS.id);
+      if (!this.el) {
+        this.el = document.createElement('div');
+        this.el.setAttribute('id', DEFAULT_OPTIONS.id);
+        this.el.classList.add('vg-tooltip');
+
+        document.body.appendChild(this.el);
+      }
+
+      const tooltipContainer = document.fullscreenElement != null ? document.fullscreenElement : document.body;
+      tooltipContainer.appendChild(this.el);
+
+      // hide tooltip for null, undefined, or empty string values
+      if (value == null || value === '') {
+        this.el.classList.remove('visible', `${DEFAULT_OPTIONS.theme}-theme`);
+        return;
+      }
+
+      // set the tooltip content
+      this.el.innerHTML = formatValue(value, DEFAULT_OPTIONS.sanitize, DEFAULT_OPTIONS.maxDepth);
```

Comment:
here you would add our own implementation of how the html would look like.

Reply:
Added.